app/views/exam_templates/_boot.js

```diff
@@ -48,39 +67,66 @@ function toggle_cover_page(id) {
   if (parsing_input.checked) {
     $("#exam-cover-display-" + id).css("display", "flex");
     attach_crop_box(id);
+    toggle_crop_scale_buttons(id);
   } else {
     $("#exam-cover-display-" + id).css("display", "none");
   }
 }
 
 function attach_crop_box(id) {
-  var jcrop_api;
+  const form = document.getElementById(`add_fields_exam_template_form_${id}`);
+  const crop_target = form.getElementsByClassName("crop-target")[0];
+
+  config_jcrop_api(crop_target, form, id);
+
+  // Set crop selection if values exist.
+  set_crop_selection(crop_target, form, id);
+}
 
+function toggle_crop_scale_buttons(id) {
   const form = document.getElementById(`add_fields_exam_template_form_${id}`);
   const crop_target = form.getElementsByClassName("crop-target")[0];
 
-  $(crop_target).Jcrop(
-    {
-      onChange: pos => {
-        const stageHeight = parseFloat(
-          getComputedStyle(crop_target, null).height.replace("px", "")
-        );
-        const stageWidth = parseFloat(getComputedStyle(crop_target, null).width.replace("px", ""));
-        const {x, y, w, h} = pos;
-
-        form.elements[`${id}_exam_template_crop_x`].value = x / stageWidth;
-        form.elements[`${id}_exam_template_crop_y`].value = y / stageHeight;
-        form.elements[`${id}_exam_template_crop_width`].value = w / stageWidth;
-        form.elements[`${id}_exam_template_crop_height`].value = h / stageHeight;
-      },
-      keySupport: false,
-    },
-    function () {
-      jcrop_api = this;
+  $("#decrease-crop-scale").on("click", function () {
+    if (crop_scale - SCALE_CHANGE < MIN_SIZE) {
+      crop_scale = MIN_SIZE;
+    } else {
+      crop_scale -= SCALE_CHANGE;
     }
-  );
 
-  // Set crop selection if values exist.
+    jcrop_api.destroy();
```

Comment:
In terms of function design, global variables are always a bit of a red flag. In this case it's okay that you're using it, but to minimize the reach/scope you can try the following:

1. Isolate the combination of `destroy` -> `config_jcrop_api` -> `set_crop_selection` into `attach_crop_box`, and call that function here and in the `increate-crop-scale` onclick. Inside `attach_crop_box`, you can guard the `destroy` call with a `if jcrop_api !== undefined` type of check.
2. Make `config_jcrop_api` *return` the Jcrop object, and then assign to `jcrop_api` inside `attach_crop_box`.
3. Modify `set_crop_selection` to take `jcrop_api` as an argument.

If I'm looking at the code correctly, this should ensure that only `attach_crop_box` really interacts with the global `jcrop_api` variable directly.